inventory/routes/concepts/pull_push_rules.rst

```diff
@@ -0,0 +1,101 @@
+===========================
+What are Pull & Push Rules?
+===========================
+
+| In inventory management, the supply chain strategy determines when
+  products should be fabricated, delivered to distribution centers and
+  made available in the retail channel.
+
+| Under a pull supply chain, actual customer demand drives the process,
+  while push strategies are driven by long-term projections.
+
+Pull Inventory Control System
+=============================
+| The *procurement inventory control system* begins with a customer's
+  order. With this strategy, companies only produce enough to fulfill their
+  customer's orders. One advantage is that there will be no
+  extra goods that needs to be stored, thus reducing inventory levels
+  and the cost of carrying and storing. However, one major
+  disadvantage to the pull system is that it is highly possible to run
+  into ordering dilemmas, such as a supplier not being able to get a
+  shipment out on time. This leaves the company unable to fulfill the
+  order and contributes to customer dissatisfaction.
+
+| An example of a pull inventory control system is the make-to-order. The
+  goal is to keep inventory levels to a minimum by only having enough
+  inventory to meet customer's demand. The :abbr:`MTO (Make-to-order)` system
+  eliminates waste by reducing the amount of storage space needed for
+  inventory and the costs of storing goods.
+
+Push Inventory Control System
+=============================
+
+| The *push system of inventory control* involves forecasting inventory
+  needs to meet customer's demand. Companies must predict which products
+  customers will purchase along with determining what quantity of goods
+  will be purchased. The company will, in turn, produce enough product to
+  meet the forecast demand and sell, or push, the goods to the consumer.
+  Disadvantages of the push inventory control system are that forecasts
+  are often inaccurate as sales can be unpredictable and vary from one
+  year to the next, but it also increases the company's costs for storing
+  these goods if the minimum stock isn’t well defined. An advantage of the push system is that the company is fairly assured it will have enough product on hand to complete customer orders, preventing the inability to meet customer demand for the product.
+  
+| A push flow indicates how locations are chained with other ones. As soon
+  as a given quantity of products is moved in the source location, a
+  chained move is automatically foreseen according to the parameters set
+  on the flow specifications (destination location, delay, type of move).
+  It can be triggered automatically or manually.
+
+Configuration
+=============
+
+| Pull and push rules are part of the routes. Go to the *Inventory
+  application, :menuselection:`Inventory --> Configuration --> Settings` and tick *Multi-Step Routes*.
```

Comment:
Broken :menuselection:
it might be because of the *italic*.
Bob likes to build, be like Bob :D